Gems/ROS2/Code/Include/ROS2/Camera/CameraSensor.h

```diff
@@ -19,6 +19,7 @@ namespace ROS2
 {
     struct CameraConfiguration
     {
+        float m_horizontalFieldOfViewDeg = 90.0f; //!< Horizontal field of view of camera sensor.
```

Comment:
This is a computed field. If this is needed as a read info, instead of a member field, please use a getter function that computes it from the 3 other parameters.

Reply:
Right! - changed